tsconfig.json

```diff
@@ -21,9 +21,5 @@
     "downlevelIteration": true,
     "target": "es2015",
     "strict": true,
-  },
-  "include": [
-    "src/*.ts",
-    "test/*.ts"
```

Comment:
Do we need to do something here so the editor doesn't get confused by the ts files in testdata?

Reply:
I found that non-sandboxed Bazel TS worker was trying to read these files from the cache for every build, following https://github.com/bazelbuild/rules_nodejs/pull/756/files
error looks like
```
ERROR: /usr/local/google/home/alexeagle/Projects/tsickle/src/BUILD:5:1: Compiling TypeScript (devmode) //src:src failed (Exit 1)
Compilation failed Error: missing input digest for /usr/local/google/home/alexeagle/.cache/bazel/_bazel_alexeagle/c5eeaf028cc4e9cd3f3c738037c2514c/execroot/tsickle/test/ast_printing_transform.ts.(only have [MEGABYTES])
```

couple options:
- turn on `--worker_sandboxing` to prevent discovering files outside the compilation unit
- use exclude rather than include in the tsconfig

Reply:
Another option would be to continue disabling worker mode in the `.bazelrc` to match old behavior, but it makes the Dev RTT slow of course

Reply:
I went with the first option and force-pushed a new commit PTAL

Reply:
(note that Bazel 0.27 introduced a new action strategy where workers get enabled by default with no settings needed in .bazelrc)